### DISCUSSION TITLE
QDate - (de)selecting multiple shifts to first date

### DISCUSSION THREAD

**Discussion by RobbeVorsselmans** [quasarframework/quasar#11170] [General - Components / Directives / etc] [answered]
Hi all

I just updated to Quasar v2 (2.2.2), but noticed some strange behaviour in the QDate component. Every time I select or deselect a date, the view shifts to the month of the first date in the model. This doesn't happen in Quasar v1, and is really annoying when you try to select multiple dates. I realize this is helpfull when you deselect a date in a far off range, but it doesn't seem beneficial to me in other cases.

The problem is visible in the default example in the documentation. [https://quasar.dev/vue-components/date#multiple-selection](url)
Just select a date in the current and next month and see how the view shifts.

Is this intentional, and if so: how do I turn this off?

Thanks in advance!
Robbe

**Comment by rstoenescu** [comment, marked as answer]
There has been logged a bug about this and I fixed it. Will be available in Quasar v2.2.3

**Reply by TomWawIT** [reply]
@rstoenescu looks like the issue still occurs in v2.12.0
https://github.com/quasarframework/quasar/discussions/17544

**Reply by TomWawIT** [reply]
open bug: https://github.com/quasarframework/quasar/issues/15637

**Reply by rstoenescu** [reply]
Discovered the root cause in latest Quasar.
Fix will be available in Quasar v2.17.7
